.github/workflows/tagged-release.yml

```diff
@@ -38,3 +44,9 @@ jobs:
         with:
           files: |
             build/release/darkreader-*
+
+      - name: Upload built artifact
+        uses: actions/upload-artifact@v2
+        with:
+          path: build/release/darkreader-*
+          retention-days: ${{ inputs.retention_days }}
```

Comment:
Does the uploading of a artifact work properly? Commits like these are only accessible by linking them or digging into /actions

Reply:
Yes, it uploads a single file named `artifact.zip` which contains all Darkreader builds. I tested this by pushing this commit to my own `master` and then dispatching this workflow from "Actions" tab.